_adr/92/index.adoc

```diff
@@ -0,0 +1,102 @@
+---
+num: 91
+title: "Toward a more unified UX: BFF"
+status: "Draft"
+authors:
+  - "Andrea Peruffo"
+  - "Riccardo Forina"
+  - "Eric Wittmann"
+tags:
+  - UI
+  - UX
+  - Dev Exp
+applies_padrs: # What PADRs does this ADR apply?
+applies_patterns: # What APs does this ADR apply?
+---
+
+## Context and Problem Statement
+
+The integration of the Enterprise/Hybrid option of RHOSAK in UI, SDKs and CLI has shown that there is a margin for improvement in how we expose and consume our services APIs.
+
+* the current services are offering the bare functionality leaving room for integration code to live in the services' consumer codebases
+* the data aggregation logic is duplicated, scattered, and eventually diverging in various codebases (especially UI and CLI)
+* there is no "single source of truth" for data aggregated from various services
+* the current REST API layer exposes the user to the entire complexity of the underlying infrastructure
```

Comment:
Is it the complexity of the infrastructure that's exposed or is it the complexity of the main service PLUS the supporting services (e.g. AMS)?  Infrastructure makes me think we're exposing network routing and whatnot.  That may be true for RHOSAK Enterprise maybe?

Reply:
Renaming `infrastructure` -> `architecture`.
We do not expose (much)"infrastructure" detail, but the user really needs to know the architecture of the system to navigate it.